src/System.Drawing.Primitives/ref/System.Drawing.Primitives.cs

```diff
@@ -7,7 +7,6 @@
 
 namespace System.Drawing
 {
-    [System.SerializableAttribute]
```

Comment:
Yes, but this is a reference assembly, which isn't really supposed to have these attributes.